constraints.pro

```diff
@@ -241,24 +278,24 @@ gen_enforced_dependency(WorkspaceCwd, DependencyIdent, null, DependencyType) :-
   workspace_has_dependency(WorkspaceCwd, DependencyIdent, DependencyRange, DependencyType),
   DependencyType == 'devDependencies'.
 
-% eth-query has an unlisted dependency on babel-runtime, so that package needs
-% to be present if eth-query is present.
-gen_enforced_dependency(WorkspaceCwd, 'babel-runtime', '^6.26.0', DependencyType) :-
-  workspace_has_dependency(WorkspaceCwd, 'eth-query', _, DependencyType).
-
-% "engines.node" must be ">=14.0.0" for all packages.
+% All packages must specify a minimum Node version of 14.
 gen_enforced_field(WorkspaceCwd, 'engines.node', '>=14.0.0').
 
-% "publishConfig.access" must be "public" for workspace packages and unset
-% for the root.
+% All published packages are public.
 gen_enforced_field(WorkspaceCwd, 'publishConfig.access', 'public') :-
   \+ workspace_field(WorkspaceCwd, 'private', true).
+% Non-published packages do not need to specify their published access level.
```

Comment:
We should consider disallowing `publishConfig` altogether for private packages, to prevent an empty `publishConfig` entry, just as with `bugs`.

Reply:
Makes sense! Fixed in [169622b](https://github.com/MetaMask/controllers/pull/1032/commits/169622ba8cd26a2a863ccf71578703fd0af9725e).